### PR TITLE
DOCSUP-5930: Update the system.part_log system table

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #18644

### CODE REVIEW

File: docs/ru/operations/system-tables/part_log.md [hunk 1]
```diff
@@ -6,29 +6,62 @@
...
--   `read_bytes` (UInt64) — сколько было прочитано байт при слиянии кусков.
--   `error` (UInt16) — код ошибки, возникшей при текущем событии.
--   `exception` (String) — текст ошибки.
+-   `event_date` ([Date](../../sql-reference/data-types/date.md)) — дата события.
+-   `event_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — время события.
+-   `duration_ms` ([UInt64](../../sql-reference/data-types/int-uint.md)) — длительность.
+-   `database` ([String](../../sql-reference/data-types/string.md)) — имя базы данных, в которой находится кусок.
+-   `table` ([String](../../sql-reference/data-types/string.md)) — имя таблицы, в которой находится кусок.
+-   `part_name` ([String](../../sql-reference/data-types/string.md)) — имя куска.
+-   `partition_id` ([String](../../sql-reference/data-types/string.md)) — идентификатор партиции, в которую был добавлен кусок. В столбце будет значение ‘all’, если таблица партициируется по выражению `tuple()`.
```
Comment (adevyatova) [inline docs/ru/operations/system-tables/part_log.md:23]:
```suggestion
-   `partition_id` ([String](../../sql-reference/data-types/string.md)) — идентификатор партиции, в которую был добавлен кусок. В столбце будет значение `all`, если таблица партициируется по выражению `tuple()`.
```

File: docs/ru/operations/system-tables/part_log.md [hunk 2]
```diff
@@ -6,29 +6,62 @@
...
+-   `path_on_disk` ([String](../../sql-reference/data-types/string.md)) — абсолютный путь к папке с файлами кусков данных.
+-   `rows` ([UInt64](../../sql-reference/data-types/int-uint.md)) — число строк в куске.
+-   `size_in_bytes` ([UInt64](../../sql-reference/data-types/int-uint.md)) — размер куска данных в байтах.
+-   `merged_from` ([Array(String)](../../sql-reference/data-types/array.md)) — массив имён кусков, из которых образован текущий кусок в результате слияния (также столбец заполняется в случае скачивания уже смерженного куска).
+-   `bytes_uncompressed` ([UInt64](../../sql-reference/data-types/int-uint.md)) — количество прочитанных разжатых байт.
```
Comment (adevyatova) [inline docs/ru/operations/system-tables/part_log.md:28]:
```suggestion
-   `bytes_uncompressed` ([UInt64](../../sql-reference/data-types/int-uint.md)) — количество прочитанных не сжатых байт.
```

File: docs/ru/operations/system-tables/part_log.md [hunk 3]
```diff
@@ -6,29 +6,62 @@
...
+-   `peak_memory_usage` ([Int64](../../sql-reference/data-types/int-uint.md)) — максимальная разница между выделенной и освобождённой памятью в контексте потока.
+-   `error` ([UInt16](../../sql-reference/data-types/int-uint.md)) — код ошибки, возникшей при текущем событии.
+-   `exception` ([String](../../sql-reference/data-types/string.md)) — текст ошибки.
 
 Системная таблица `system.part_log` будет создана после первой вставки данных в таблицу `MergeTree`.
 
+**Пример**
+
+``` sql
+SELECT * FROM system.part_log LIMIT 1 \G
```
Comment (adevyatova) [inline docs/ru/operations/system-tables/part_log.md:40]:
```suggestion
SELECT * FROM system.part_log LIMIT 1;
```
\G лишнее здесь?

File: docs/en/operations/system-tables/part_log.md [hunk 4]
```diff
@@ -6,29 +6,62 @@ This table contains information about events that occurred with [data parts](../
...
--   `read_bytes` (UInt64) — The number of bytes was read during the merge.
--   `error` (UInt16) — The code number of the occurred error.
--   `exception` (String) — Text message of the occurred error.
+-   `event_date` ([Date](../../sql-reference/data-types/date.md)) — Event date.
+-   `event_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — Event time.
+-   `duration_ms` ([UInt64](../../sql-reference/data-types/int-uint.md)) — Duration.
+-   `database` ([String](../../sql-reference/data-types/string.md)) — Name of the database the data part is in.
+-   `table` ([String](../../sql-reference/data-types/string.md)) — Name of the table the data part is in.
+-   `part_name` ([String](../../sql-reference/data-types/string.md)) — Name of the data part.
+-   `partition_id` ([String](../../sql-reference/data-types/string.md)) — ID of the partition that the data part was inserted to. The column takes the ‘all’ value if the partitioning is by `tuple()`.
```
Comment (adevyatova) [inline docs/en/operations/system-tables/part_log.md:23]:
```suggestion
-   `partition_id` ([String](../../sql-reference/data-types/string.md)) — ID of the partition that the data part was inserted to. The column takes the `all` value if the partitioning is by `tuple()`.
```

**Comment by sevirov** [comment]
@akuzm Please, review this PR.

**Pull request merged**